appveyor.yml

```diff
@@ -1,6 +1,6 @@
 version: dev.{build}
 skip_tags: true
-image: Visual Studio 2019 Preview
+image: Visual Studio 2022 Preview
```

Comment:
Looks like you also need to remove " Preview"

Reply:
Perfect, thank you